### PR TITLE
SystemUI: Add restart SystemUI in Advanced Reboot [1/2]

### PR DESCRIPTION
**Pull request by awakened1712**
Tested, working nicely.

### CODE REVIEW

**Comment by gwolf2u** [comment]
please rebase

**Comment by awakened1712** [comment]
> please rebase

I'm on holiday. I'll do when I'm back next Monday

**Comment by ShevT** [comment]
What is it for?

**Comment by gwolf2u** [comment]
> What is it for?

restart SystemUI in reboot menu
there are some use cases for it I guess

**Comment by ShevT** [comment]
> restart SystemUI in reboot menu there are some use cases for it I guess

I not see practical need

**Comment by gwolf2u** [comment]
> > restart SystemUI in reboot menu there are some use cases for it I guess
> 
> I not see practical need

Example:
Some weird UI glitch that fixes itself after a reboot
This case simply restart systemui

**Comment by ShevT** [comment]
> Example: Some weird UI glitch that fixes itself after a reboot This case simply restart systemui

Ok

**Comment by awakened1712** [comment]
> please rebase

Rebased

File: packages/SystemUI/res/values/cm_strings.xml [hunk 1]
```diff
@@ -34,6 +34,8 @@
     <string name="global_action_restart_download">Download</string>
     <!-- Button to restart the device into fastboot mode, within the Restart Options dialog -->
     <string name="global_action_restart_fastboot">Fastbootd</string>
+    <!-- Button to restart SystemUI, within the Restart Options dialog -->
+    <string name="global_action_restart_systemui">SystemUI</string>
```
Comment (gwolf2u) [inline packages/SystemUI/res/values/cm_strings.xml:38]: move string to cr_strings.xml

**Review by neobuddy89 — COMMENTED** [review]
@awakened1712 move string to cr_strings.xml in the end and not cm_strings.xml

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by awakened1712** [comment]
> @awakened1712 move string to cr_strings.xml in the end and not cm_strings.xml

Done

**Comment by ShevT** [comment]
[ic_restart_systemui.xml](https://github.com/crdroidandroid/android_frameworks_base/pull/947/commits/da8472d988c1e84d80190f72ee4fb808147ec2be#diff-4a31dcdb5b23836530469ac888bc6a8f1802900c3f4fd1b589dae58ecd832f94) no newline at end of file.

**Comment by awakened1712** [comment]
> [ic_restart_systemui.xml](https://github.com/crdroidandroid/android_frameworks_base/pull/947/commits/da8472d988c1e84d80190f72ee4fb808147ec2be#diff-4a31dcdb5b23836530469ac888bc6a8f1802900c3f4fd1b589dae58ecd832f94) no newline at end of file.

Added

**Review by neobuddy89 — APPROVED** [review]

**Pull request merged**